Yggdrasil/src/app/midi/server/start-midi-server.ts

```diff
@@ -159,6 +160,7 @@ const startMidiServer = async () => {
                     signalPixelblaze(0, 0)
                     signalPixelblaze(1, 0)
                     signalPixelblaze(2, 0)
+                    break
```

Comment:
@AlexMakiJokela Was there a functionality reason why we did not have breaks for some cases in this switch statement? Should they instead been included in the next or previous cases?